src/wallet/parser.rs

```diff
@@ -212,7 +227,19 @@ mod tests {
     fn from_raw(raw_inscriptions: Vec<Vec<u8>>) -> OrdResult<Vec<OrdParser>> {
         raw_inscriptions
             .into_iter()
-            .map(|inscription| OrdParser::categorize(&inscription))
+            .map(|inscription| {
+                let nft = Nft {
```

Comment:
Some `Nft` inscriptions may actually contain other fields such as `metaprotocol` (e.g. BRC20, Bitmap, etc). Deriving default for the other fields will overwrite them.